### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Debugging-Console_Color Contrast: Color contrast ratio for gray color text is 3.9:1 which is < 4.5:1

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3592]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ColorContrast; #MAS1.4.3; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

MAS Violated: 1.4.3

**Repro Steps:**
1)Launch IntelliJ IDEA in light theme
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till <Project Name>-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Tab till Edit Configurations menu(select the icon in the upper-right corner)
7)Navigate to Advanced Configuration (Remote Debugging) section and Press the shortcut key to check Enable Spark remote debug
8)Fill Serure Shell (SSH) User Name and SSH password
9)Sent some breakpoints in SparkCore_wasbloTest code and tab till sparkjobdebug and press enter
10)Press cntrl+tab to navigate to Debug Remote Spark Job in Cluster section and verify the color contrast ratio for gray color text in both debugger & console tabs

**Actual Result:**
 Color contrast ratio for gray color text is 3.9:1 which is < 4.5:1

**Expected Result:**
 Color contrast ratio for gray color text should be >= 4.5:1

**User Impact:**
Low vision users will find difficult to read the content if the text does not meet the standard color contrast ratio

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![Bug_3592_color contrast1](https://user-images.githubusercontent.com/42638450/65419391-2a33d000-de1c-11e9-9a66-298928aef867.png)
![Bug_3592_color contrast2](https://user-images.githubusercontent.com/42638450/65419393-2a33d000-de1c-11e9-88af-cc114cb389f2.png)

**Comment by t-rufang** [comment]
The debug window is supported by IntelliJ.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749695
